stl/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@
 
 set(HEADERS
     ${CMAKE_CURRENT_LIST_DIR}/inc/__msvc_all_public_headers.hpp
+    ${CMAKE_CURRENT_LIST_DIR}/inc/__msvc_int128.hpp

```

Comment:
I've copied this list into the OP where GitHub won't bury it under "372183921 hidden conversations". (It would be nice to draw attention to the three unresolved conversations amid the 372183918 resolved ones, GitHub!)